src/Microsoft.Health.Dicom.Core/Features/Export/IdentifierExportOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+using Microsoft.Health.Dicom.Core.Models.Common;
+
+namespace Microsoft.Health.Dicom.Core.Features.Export;
+
+internal sealed class IdentifierExportOptions : IValidatableObject
+{
+    public IReadOnlyList<string> Values { get; set; }
+
+    public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+    {
+        var results = new List<ValidationResult>();
+        if (Values == null || Values.Count == 0)
+        {
+            results.Add(new ValidationResult(string.Format(CultureInfo.CurrentCulture, DicomCoreResource.MissingProperty, nameof(Values)), new string[] { nameof(Values) }));
+        }
+        else
+        {
+            try
+            {
+                foreach (string value in Values)
+                {
+                    if (value == null)
+                        throw new ValidationException(string.Format(CultureInfo.CurrentCulture, DicomCoreResource.InvalidDicomIdentifier, value));
+
+                    DicomIdentifier.Parse(value);
```

Comment:
nit: generaly not a fan of catching and wrapping exception. They should be used to bubble up problems. Does that mean we need a version of this that returns parse status with a message?

Reply:
I tried to look at `Parse` in isolation and thought that a failed parse would return `FormatException` like `DateTime.Parse`. And then when we call it here, we wrap that exception in our own `ValidationException` used by this API. I think this is generally a good idea, but here in this case I could easily write a `TryParse` method instead and use its result to throw